docs/README.md

```diff
@@ -1,503 +1,491 @@
 
 <br>
 
-Elements can post events its parent container (your website) when a shopper interacts with the checkout element. Here are the events it can post:
-
-| Event Code                  | Purpose                                                                                                                                                                                  |
-| --------------------------- | ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
-| onReady                     | The event fires when a given element resource has loaded.                                                                                                                                |
-| onSubmit                    | The event is raised when confirm the intent. It fires after the click Pay button or calling confirmPaymentIntent function.                                                               |
-| onSuccess                   | The event fires when a intent is confirm with Airwallex                                                                                                                                  |
-| onError                     | Error events are fired at various targets for different kinds of errors with shopper interaction. See error codes [here](https://www.airwallex.com/docs/api?v=2019-09-09#/Errors).       |
-| onCancel                    | The event fires when shopper click cancel button when interact with the payment form.                                                                                                    |
-| onFocus                     | The event is raised when the shopper sets focus on an input by click or tab switch interaction.                                                                                          |
-| onBlur                      | The event is raised when an input in element loses focus.                                                                                                                                |
-| onChange                    | The events fire when the user commits a value change to a input. This may be done, for example, by clicking outside of the input or by using the Tab key to switch to a different input. |
-| onClick                     | The event is raised when the user clicks on an input element.                                                                                                                            |
-| onDynamicCurrencyConversion | The events fire when merchant enable Dynamic Currency Conversion (DCC) feature and shopper is confirm payment with an intent which match DCC scenario                                    |
+### Get an Element
+
+This method looks up a previously created Element by its type.  
+
+```ts
+Airwallex.getElement(type)
```

Comment:
same above, if customer is using npm, the code will be the following:

import { getElement } from '@airwallex/airwallex-payment-elements';
const element = getElement(type)

Reply:
Updated.